Test/CaptureFileReaderTest.cs

```diff
@@ -11,16 +11,49 @@ public class CaptureFileReaderDeviceTest
     {
         private static int capturedPackets;
 
+        [Test]
+        public void CaptureNonExistant()
+        {
+            Assert.Throws<PcapException>(() =>
```

Comment:
I expected to see a `FileNotFoundException`

Reply:
Interesting huh? Should we check to see if the file exists as a first test when creating a capture file reader? That would let us report a file not found exception.

Reply:
I see that `pcap_open_offline` does not return an error code, only a message.

So the exception type is understandable.

Reply:
We could easily check to see if the file exists no? Then we could return the more typical exception in the common case of the file not being found and reserve the PcapException for other error conditions. Let me make that change as well in another commit.